### DISCUSSION TITLE
how do I change the icon color of the toolbar button?

### DISCUSSION THREAD

**Discussion by jianyunli** [vitmalina/w2ui#2176] [Q&A] [answered]
I tried use jQuery .css() function, doesn't work. also set the class itself or add extra class, none works

.fa-wrench {
  color: black;
}

.blackcolor {
  color: red;
}

.w2ui-tb-icon{
  color: red;
}

**Comment by mpf82** [comment, marked as answer]
``item.style`` is only applied to the caption: https://w2ui.com/web/docs/2.0/w2toolbar.item_template

Since the icons already have a color set by the w2ui css, you must either define a very specific selector or use ``!important``

Example: 

```
.w2ui-tb-icon > span {
  color: green !important;
}
.w2ui-tb-icon > span.w2ui-icon-plus {
  color: blue !important;
}
```

Fiddle: 

http://jsfiddle.net/2pnudbwy/

**Reply by jianyunli** [reply]
Perfect, thank you so much for the quick reply.

by the way, for other people's use, I used font awesone icon, small changes for me
`.w2ui-tb-icon > span.fa-plus {
   color: blue !important;
}`
